pkg/block/lsblk/lsblk.go

```diff
@@ -52,14 +53,15 @@ type jParts []*block.Partition
 func (p jPart) Partition() *block.Partition {
 	// Converts B to MB
 	return &block.Partition{
-		FilesystemLabel: p.Label,
-		Size:            uint(p.Size / (1024 * 1024)),
-		FS:              p.FS,
-		Flags:           []string{},
-		MountPoints:     p.MountPoints,
-		Path:            p.Path,
-		Disk:            p.Disk,
-		Name:            p.Name,
+		Label:       p.Label,
+		Size:        uint(p.Size / (1024 * 1024)),
+		FileSystem:  p.FS,
+		UUID:        p.UUID,
```

Comment:
This is the filesystem UUID which could be used in fstab as `UUID=afadfa` to indentify the device.